data/Cargo.toml

```diff
@@ -6,11 +6,16 @@ name = "flip-flop-data"
 version = "0.1.0"
 
 [dependencies]
-serde = { version = "1.0.126", default-features = false }
+aead = { version = "0.5", default-features = false }
+heapless = "0.7"
+postcard = "1.0"
+rand = { version = "0.8", default-features = false }
+serde = { version = "1.0", default-features = false }
```

Comment:
I've moved some of the dev dependencies into here as the data layer is quite prescriptive in terms of how it should be used e.g. AES CCM 128 bit encryption, 7 byte nonce, 4 byte MIC, postcard wire format... I wanted to eliminate heapless but then found myself re-writing it. :-)